rolling/distribution.yaml

```diff
@@ -7588,25 +7527,6 @@ repositories:
       url: https://github.com/ros-drivers/velodyne.git
       version: ros2
     status: developed
-  velodyne_simulator:
-    doc:
-      type: git
-      url: https://bitbucket.org/DataspeedInc/velodyne_simulator.git
-      version: foxy-devel
-    release:
-      packages:
-      - velodyne_description
-      - velodyne_gazebo_plugins
```

Comment:
Same question here; can we just blacklist this one package, rather than removing the whole thing?